kubetest/OWNERS

```diff
@@ -4,6 +4,7 @@ approvers:
 - BenTheElder
 - krzyzacy
 - shyamjvs
+- wojtek-t # For European timezone emergencies.
```

Comment:
Doesn't someone also need ability to build/rollout kubetest changes? @krzyzacy @BenTheElder

Reply:
I think @wojtek-t should already have the ability to do this.

Reply:
Technically, if a job relies on a kubetest change, submit the kubetest PR first, never change config and kubetest at the same time :-)

Reply:
> Technically, if a job relies on a kubetest change, submit the kubetest PR first, never change config and kubetest at the same time :-)

Yes, but what I'm saying is currently how will someone from EU make these changes? That is:
- are they allowed to build+push new kubetest image?
- are they allowed to update image tags?

> I think @wojtek-t should already have the ability to do this.

@BenTheElder I don't think so.. For e.g see this change in kubekins image - https://github.com/kubernetes/test-infra/pull/11264/commits/bdf6cd2e3f906251977ea69b5780df4920a974ab. That needs approve for `images/`. Not sure if there are more like that..

Reply:
@shyamjvs you don't have to bump bootstrap every time. I bumped that because I changed something under image/bootstrap

Reply:
I'm fine with that if you say bootstrap changes won't be needed to fix broken CI. In my past experience, I needed to change it a handful of times :)

Reply:
@shyamjvs sorry for the confusion but we have

* https://github.com/kubernetes/test-infra/tree/master/images/bootstrap, which is a shared base image layer for kubekins and maybe other images

* https://github.com/kubernetes/test-infra/blob/master/jenkins/bootstrap.py - the deprecated runner, which is NOT included in the bootstrap image and should not be touched anymore

besides the naming, the two "bootstrap" in test-infra doesn't interact with each other much :-)

Reply:
if we absolutely have to get a change like that in, wojekt also still has write for now. but generally bootstrap should _not_ be bumped in urgency. that image is just a base layer and should only need changing when we upgrade eg go, bazel.

Reply:
Sounds good, thanks.

> if we absolutely have to get a change like that in, wojekt also still has write for now
Do you mean write access to this repo?

Reply:
yes https://github.com/kubernetes/test-infra/pull/11250#event-2133224387

Reply:
@BenTheElder Then why take this trouble adding to any OWNERS file at all? :)

Reply:
because long term we should be using these. I really do not think images/ should need to be touched, but if we find out that they in fact do, wojekt will be able to fix it and then we can update owners again :-)

Reply:
we get alerts for manual merges, so we'll know if that occurs. they're not meant to be used generally.

Reply:
Technically nobody should have manual merge power except for oncalls - we'll clean that part up.

Reply:
Not blocking anything wrt this PR.. but in general I'm seeing a conflict here b/w "oncalls who should be able to merge anything" vs "approvers who should be able to approve changes in their area of code".

Reply:
What is the conflict?

I would not say that oncall needs to be able to merge anything. Oncall needs to be able to restore and maintain the health of the system.

Reply:
Right. So shouldn't we then grant OWNERS to the oncall for 'all places that can break'? (which can potentially be anywhere in the repo, therefore the root OWNERS file)

Reply:
yes, see https://github.com/kubernetes/test-infra/blob/master/OWNERS#L3-L12

Reply:
Yup. But who in there is oncall for EU timezone? (which is our problem in question here)

Reply:
test-infra oncall is only 10a-6p PST and on a best-effort basis - if something is really broken at midnight PST, people should file an issue and wait for next oncall to fix them in the morning.

People from EU can own certain subdirs to accelerate their dev cycles, like certain config changes, but they should not need to own every single infra piece. Ideally they should not even own kubetest, they can push and use their own kubetest images, and wait for one of the kubetest owners to review their changes.

Reply:
As soon as people in europe join the oncall rotation we will add them

Reply:
Ok, thanks for clarifying!